pkg/controller/machinepool/machinepool_controller.go

```diff
@@ -402,7 +473,42 @@ func (r *ReconcileMachinePool) getRemoteMachineSets(
 	return remoteMachineSets, nil
 }
 
-func (r *ReconcileMachinePool) generateMachineSets(
+func (r *ReconcileMachinePool) getLocalMachineSets(
+	targetNamespace string,
+	logger log.FieldLogger,
+) (*capiv1.MachineSetList, error) {
+	localMachineSets := &capiv1.MachineSetList{}
+	if err := capiv1.AddToScheme(r.scheme); err != nil {
```

Comment:
Again, this is done [already](https://github.com/openshift/hive/pull/1459/commits/fd373abba43fdfcf9ac1e3a3afb14e732f548380#diff-08e7c6d95e969b3e8021c8956329e0e0612791c389268ae29b85ee11971f55ecR75)